code/modules/jobs/job_types/janitor.dm

```diff
@@ -14,7 +14,7 @@
 	alt_titles = list("Custodian", "Sanitation Worker", "Cleaner", "Caretaker", "Maid")
 
 	added_access = list()
-	base_access = list(ACCESS_JANITOR, ACCESS_MAINT_TUNNELS, ACCESS_MINERAL_STOREROOM)
+	base_access = list(ACCESS_JANITOR, ACCESS_MAINT_TUNNELS, ACCESS_MINERAL_STOREROOM, ACCESS_CARGO, ACCESS_RESEARCH, ACCESS_MEDICAL, ACCESS_ENGINE, ACCESS_SEC_DOORS)
```

Comment:
> Janitors should interact with members of departments to get where they need to be, or interact with the HoP to get access to these areas.

Wdym that's just a rule you made up on the spot.
If anything, it's already well established in movie and game tropes that the janitor already has access to basically everywhere because when people try to infiltrate security in games like GTA 5 or in movies like the lookout or The score

Reply:
> NEVER been that way.
So what?

Reply:
> I'm saying janitors shouldn't have access everywhere because it has NEVER been that way.

Well yeah of course it hasn't been that way.. this is why this is a PR. It is here to change it to something else.

You know the entire point of development 🫠

Reply:
ok first of all don't tell me to close my own PR.
second of all, you're spreading misinformation, these accesses are purely for department access like the medbay front doors, or research division doors, etc.

Reply:
> > Janitors should interact with members of departments to get where they need to be, or interact with the HoP to get access to these areas.
> 
> Wdym that's just a rule you made up on the spot. If anything, it's already well established in movie and game tropes that the janitor already has access to basically everywhere because when people try to infiltrate security in games like GTA 5 or in movies like the lookout or The score they disguise as a janitor to gain that access.

Bruh, you are smoking some heavy crack. The whole point janitors get in everywhere is also because no one suspects a janitor and lets them in. There is enough movies out there that basically show janitors not using keys but words to get in. Giving janitors basic access everywhere is a dumb move, literally it is up to the people to let them in. Not this pr to make this dumb fucking move and have powergaming janitors every shift.